include/extractor/profile_properties.hpp

```diff
@@ -86,6 +105,8 @@ struct ProfileProperties
     bool fallback_to_duration;
     //! stores the name of the weight (e.g. 'duration', 'distance', 'safety')
     char weight_name[MAX_WEIGHT_NAME_LENGTH + 1];
+    //! stores the names of each class
+    std::array<char[MAX_CLASS_NAME_LENGTH + 1], MAX_CLASS_INDEX + 1> class_names;
```

Comment:
Hm why not a vector of strings? Serialization and deserialization should be independent of how we store it here I think?